scripts/docker-integration-tests/prometheus/test.sh

```diff
@@ -207,6 +239,8 @@ function test_query_restrict_metrics_type {
 echo "Running prometheus tests"
 test_prometheus_remote_read
 test_prometheus_remote_write_multi_namespaces
+test_prometheus_remote_write_empty_label_name_returns_400_status_code
+test_prometheus_remote_write_duplicate_label_returns_400_status_code
```

Comment:
Worth writing graphite integration test?

Reply:
Unfortunately hard to get back a hard error from graphite since it's just writing to a port of the process with no return code per metric processed.

Reply:
true, I guess just that the metric doesn't appear?

Reply:
Would be a weird test though